houses/views.py

```diff
@@ -1,3 +1,68 @@
-from django.shortcuts import render
+import json
+
+from django.views            import View
+from django.http             import JsonResponse
+
+from .models                 import House, HouseImage, HouseType, Ghost ,Country,City
+from reservations.models     import Reservation
+
+
+
+class HouseDetailView(View):
+    def get(self, request, house_id):
+        houses = House.objects.filter(id = house_id)
+        houseimage = HouseImage.objects.filter(id=house_id)
+        house = House.objects.get(id=house_id)
```

Comment:
- 피드백 1. db에서 데이터 하나를 가져올때에는 filter 보다는 get을 사용해주세요. 
- 피드백 2. `houseimage = HouseImage.objects.filter(id=house_id)` 이 코드는 아래 아무곳에서도 사용되고 있지 않습니다. 지워주세요.

```suggestion
        houses = House.objects.get(id = house_id)
        house = House.objects.get(id=house_id)
```